solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -318,6 +323,37 @@ contract InterchainAccountRouter is Router {
         }
     }
 
+    function moduleType() external pure returns (uint8) {
+        return uint8(IInterchainSecurityModule.Types.ROUTING);
+    }
+
+    CommitmentReadIsm public immutable CCIP_READ_ISM;
```

Comment:
declare this next to the other constants

Reply:
I can do this, but I actually would like to move to having variables defined next to the functions that use them. We can discuss more offline.

Reply:
since these can only be set in the constructor I would expect them to be declared before the constructor

Reply:
Done